geode-assembly/build.gradle

```diff
@@ -216,7 +216,7 @@ dependencies {
   }
   acceptanceTestCompile(project(':geode-assembly:geode-assembly-test'))
   acceptanceTestCompile('org.apache.httpcomponents:httpclient')
-  acceptanceTestCompile('org.gradle:gradle-tooling-api:5.2.1')
+  acceptanceTestCompile('org.gradle:gradle-tooling-api:5.1.1')
```

Comment:
Please add a comment as to why this is a different version and to leave it this version when gradle is rev'd.

Reply:
Done